code/extension/Member.php

```diff
@@ -26,6 +26,8 @@ class Member extends \DataExtension
 
     private static $admins_can_disable = false;
 
+    private static $regenerate_on_activation = true;
```

Comment:
These changes from #13 have snuck in here - can we make sure 1 PR = 1 feature, helps with review and faster merges ;)

Add the cancel button only on it's own branch and alter the PR branches here on GH trying to avoid a close and new PR if possible 👍